### ISSUE TITLE
WCAG 2.0 compliance article does not clarify which level is supported

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor-dev#2893]
Moved from https://github.com/ckeditor/ckeditor-docs/issues/189.

https://ckeditor.com/docs/ckeditor4/latest/guide/dev_wcag.html doesn't define which level is met.

**Comment by f1ames** [comment]
@Reinmar (as an author of the original issue), I can see the link table covers compliance levels:

![image](https://user-images.githubusercontent.com/1061942/53742414-dd48d600-3e98-11e9-9dd4-b9d6dc126e18.png)

Didi you mean something else? How it should be described there?
